string action_id
string state

# Enum
string WAITING = "WAITING"
string INITIALIZING = "INITIALIZING"
string RUNNING = "RUNNING"
string FINISHED = "FINISHED"
string FAILED = "FAILED"
string CANCELLED = "CANCELLED"
